racket/src/ChezScheme/s/cp0.ss

```diff
@@ -906,6 +906,16 @@
                                 (make-if ctxt sc e11 (non-result-exp e12 e2) (non-result-exp e13 e3)))])))
                   #f)]
              [else #f])]
+          [(nanopass-case (Lsrc Expr) (result-exp e1)
+             [(call ,preinfo ,e10 ,e11, e12)
```

Comment:
Typo: space should be before last `,` instead of after

Reply:
fixed